modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2519,7 +2519,7 @@
 				if _ACTION >= "vs2019" then
 					min = "10.0"
 				else
-					min = iif(_ACTION == "vs2017", m.latestSDK10Version(), nil)
+					min = nil
```

Comment:
I would prefer if you just replaced `m.latestSDK10Version()` with `"$(LatestTargetPlatformVersion)"`.